### PR TITLE
Fixing bug in cloudformation command caused by missing type-checking

### PR DESCRIPTION
**Pull request by sanathkr**
Fixes https://github.com/aws/aws-cli/issues/2435

### CODE REVIEW

**Comment by stealthycoin** [comment]
cc @JordonPhillips @jamesls @kyleknap @dstufft

File: awscli/customizations/cloudformation/yamlhelper.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 # language governing permissions and limitations under the License.
 
 import yaml
+import six
```
Comment (kyleknap) [inline awscli/customizations/cloudformation/yamlhelper.py]: So for ``six`` could you import from ``awscli.compat`` instead?: https://github.com/aws/aws-cli/blob/develop/awscli/compat.py#L17 The CLI does not have a direct dependency on ``six`` which would be the case for this import, but instead relies on botocore's vendored version of ``six`` and imports that in the ``compat`` module

Comment (sanathkr) [inline awscli/customizations/cloudformation/yamlhelper.py]: Done. Pushed the change

**Comment by kyleknap** [comment]
Awesome. Merging.

**Pull request merged**